doc/assets/planning/planning_structure.drawio

```diff
@@ -0,0 +1,314 @@
+        </mxCell>
+        <mxCell id="VXw4u54tnH0GYowiBxqF-25" value="&lt;font style=&quot;font-size: 12px;&quot;&gt;Speed Limits&lt;/font&gt;" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" parent="VXw4u54tnH0GYowiBxqF-24" vertex="1" connectable="0">
+          <mxGeometry x="-0.2123" relative="1" as="geometry">
+            <mxPoint y="22" as="offset" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="VXw4u54tnH0GYowiBxqF-2" value="&lt;font style=&quot;font-size: 14px;&quot;&gt;&lt;b&gt;OpenDrive&lt;/b&gt;&lt;/font&gt;" style="ellipse;whiteSpace=wrap;html=1;fillColor=#d0cee2;strokeColor=#56517e;" parent="1" vertex="1">
+          <mxGeometry x="170" y="570" width="120" height="80" as="geometry" />
+        </mxCell>
+        <mxCell id="VXw4u54tnH0GYowiBxqF-17" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=1;exitY=0.5;exitDx=0;exitDy=0;entryX=0;entryY=0.5;entryDx=0;entryDy=0;fillColor=#e51400;strokeColor=#B20000;" parent="1" source="VXw4u54tnH0GYowiBxqF-12" target="VXw4u54tnH0GYowiBxqF-14" edge="1">
+          <mxGeometry relative="1" as="geometry">
+            <Array as="points">
+              <mxPoint x="990" y="160" />
+              <mxPoint x="990" y="360" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="VXw4u54tnH0GYowiBxqF-18" value="&lt;div&gt;&lt;i&gt;&lt;font style=&quot;font-size: 12px;&quot;&gt;Target Velocity and Trajectory &lt;/font&gt;&lt;/i&gt;&lt;/div&gt;(/trajectory)" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" parent="VXw4u54tnH0GYowiBxqF-17" vertex="1" connectable="0">
+          <mxGeometry x="-0.4409" y="1" relative="1" as="geometry">
+            <mxPoint x="-1" y="68" as="offset" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="vDj7rED1BERV3GODybnN-17" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=0;exitY=0.5;exitDx=0;exitDy=0;entryX=1;entryY=0.5;entryDx=0;entryDy=0;startArrow=classic;startFill=1;fillColor=#fad7ac;strokeColor=#b46504;" edge="1" parent="1" source="VXw4u54tnH0GYowiBxqF-12" target="VXw4u54tnH0GYowiBxqF-13">
+          <mxGeometry relative="1" as="geometry">
+            <Array as="points">
+              <mxPoint x="660" y="170" />
+              <mxPoint x="660" y="180" />
+              <mxPoint x="550" y="180" />
+              <mxPoint x="550" y="160" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="VXw4u54tnH0GYowiBxqF-12" value="&lt;div&gt;&lt;font style=&quot;font-size: 13px;&quot;&gt;&lt;b&gt;Motion Planning&lt;/b&gt;&lt;/font&gt;&lt;/div&gt;&lt;div&gt;&lt;font style=&quot;font-size: 12px;&quot;&gt;- Take most disposable Information to make final trajectory Decision&lt;/font&gt;&lt;/div&gt;" style="ellipse;whiteSpace=wrap;html=1;fillColor=#f5f5f5;strokeColor=#666666;gradientColor=#b3b3b3;" parent="1" vertex="1">
```

Comment:
I’m not sure what you mean by ‚disposable information‘. Can you give me a bit more detail please?

Reply:
It meant that the node takes most available information into account when generating the desired trajectory and velocity. 
I changed the description, it should be less confusing...